alembic/versions/82ed8fc51745_add_course_sandbox_flag.py

```diff
@@ -0,0 +1,21 @@
+"""add course sandbox flag
+
+Revision ID: 82ed8fc51745
+Revises: 346c3877ffae
+Create Date: 2017-10-10 11:55:58.012318
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '82ed8fc51745'
+down_revision = '346c3877ffae'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('course', sa.Column('sandbox', sa.SmallInteger(), nullable=False, server_default='0', default=0));
```

Comment:
its saver use `with op.batch_alter_table('course', naming_convention=convention) as batch_op:`
for both upgrades and downgrades (especially downgrades since sqlite frequently doesn't like when you drop columns without doing a batch alter in the backend tests)

Reply:
Also please add an index to the column (since it will be used for filtering)